### PR TITLE
Tests fix

### PR DESCRIPTION
**Pull request by mesmith75**
The latest LbScripts uses gcc8 which is throwing a fit when compiling psutil.

Although that is a worry as we shouldn't be relying on compilers...

### CODE REVIEW

**Review by egede — COMMENTED** [review]
The dependency is created by the `memory_profiler` dependency added by @ishanrai05 . As we do not want this to be run by all users, we can maybe think about implementing this in a different way (like the optional dependency we have for LHCb now).

The actual problem is that the wrong gcc is used for building. If I keep the gcc8 configuration and do 
`source /cvmfs/lhcb.cern.ch/lib/lcg/releases/LCG_95/gcc/8.2.0/x86_64-centos7/setup.sh`
it works for me. Obviously that is not a great solution either.

**Comment by egede** [comment]
Maybe to solve the problem short-term and get the testing running again, just remove memory_profile from the dependencies. That can then be fixed by solving #1483.

**Comment by mesmith75** [comment]
I think the importing of the modules needs to be moved as well.

**Comment by egede** [comment]
@ishanrai05 Can you move the import of `memory_profile` such that `ganga` can run without `memory_profile` installed if it is not enabled in the `.gangarc` file. The best will be to do it on this PR such that it confirms that the testing is working again.

**Comment by ishanrai05** [comment]
Ref : #1483

File: setup.py [hunk 1]
```diff
@@ -90,6 +90,9 @@ def run(self):
           'absl-py>=0.1.2',
           'google-api-python-client>=1.1',
       ],
+      extra_require=[
+          'memory_profiler',
+      ],
```
Comment (mesmith75) [inline setup.py]: This needs to be a dict with the name of the option and then a list of the requirements.
i.e.
```python
extra_require={'profiler' : ['memory_profiler']}
```

**Comment by mesmith75** [comment]
You also need to move the places the module is imported until after you have checked if you are running the profiling.

File: setup.py [hunk 2]
```diff
@@ -90,6 +90,7 @@ def run(self):
           'absl-py>=0.1.2',
           'google-api-python-client>=1.1',
       ],
+      extra_require={'profiler' : ['memory_profiler']},
```
Comment (alexanderrichards) [inline setup.py:93]: Just question to @mesmith75 didn't we already have an extra_require for LHCb? What happened to that?

**Review by alexanderrichards — APPROVED** [review]

File: setup.py (hunk 2 above)
Comment (mesmith75) [reply setup.py:93]: Its in another PR - #1462

Comment (alexanderrichards) [reply setup.py:93]: thanks

**Pull request merged**